dune_client/client_async.py

```diff
@@ -135,6 +145,14 @@ async def get_result(self, job_id: str) -> ResultsResponse:
         except KeyError as err:
             raise DuneError(response_json, "ResultsResponse", err) from err
 
+    async def get_latest_result(self, query: Query) -> ResultsResponse:
+        """GET latest results from Dune API for `query_id`"""
+        response_json = await self._get(url=f"query/{query.query_id}/results")
```

Comment:
It looks like Query object isn't even necessary here (just the ID). I am trying to understand how to fetch the latest result for a given set or parameters. Does not seem to be part of this, but I am pretty sure this works in the interface.

Reply:
Thanks for noticing that. I'll check the API behaviour in case of non-empty query params.

Reply:
Committed the update. Not sure if it's ok, can you check @bh2smith?

Reply:
In case if there is no result for specified set of params it will return 

`DuneError: Can't build ResultsResponse from {'error': 'not found: No execution found for the latest version of the given query'}`